crates/bevy_ui/src/entity.rs

```diff
@@ -230,6 +231,95 @@ impl Default for ButtonBundle {
         }
     }
 }
+
+/// A UI node that is a slider
+#[derive(Bundle, Clone, Debug, Default)]
+pub struct SliderBundle {
```

Comment:
Both these bundles and their related system should belong to a `bevy_widget` module and not `bevy_ui`, see the approach of #6517